### PR TITLE
Docs: Add katana, Starknet devnet and Starknet Foundry installation

### PR DESCRIPTION
**Pull request by gianalarcon**
Hi team

Since I am working on a [katana tutorial and starknet fundry tutorial](https://github.com/starknet-io/starknet-docs/pull/1191) for starknet-docs.
I suggest to add the katana, Starknet devnet and Starknet Foundry installation on this section

Please check @stoobie

### CODE REVIEW

File: modules/ROOT/partials/partial_environment_setup.adoc [hunk 1]
```diff
@@ -122,3 +127,39 @@ scarb --version
...
 Scarb should now be installed.
+
+[#installing_katana]
+== Installing Katana
+
+The steps for installing Katana and upgrading Katana are identical.
+
+.Procedure
+
+. Install dojoup, the installer for the Katana toolchain:
```
Comment (stoobie) [inline modules/ROOT/partials/partial_environment_setup.adoc]:
```suggestion
. Install Dojoup, the installer for the Katana toolchain:
```

File: modules/ROOT/partials/partial_environment_setup.adoc [hunk 2]
```diff
@@ -122,3 +127,39 @@ scarb --version
...
+Katana should now be installed.
+
+. Restart the terminal and run the following command to verify the installation:
++
+[source,shell]
+----
+katana --version
+----
++
+katana prints the current version.
```
Comment (stoobie) [inline modules/ROOT/partials/partial_environment_setup.adoc]:
```suggestion
Katana prints the current version.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
A couple of minor suggestions.

I think we should wait to merge this until I finish reviewing the other PR.

**Review by gianalarcon — COMMENTED** [review]
Hi @stoobie. Yes, That's fine, we can wait for the other PR to be reviewed

File: modules/ROOT/partials/partial_devtools.adoc [hunk 3]
```diff
@@ -115,7 +115,7 @@ With starknet-devnet-rs, you can do the following:
 link:https://github.com/0xSpaceShard/starknet-devnet-rs[The starknet-devnet-rs Github repository]
 
 
-[#katana]
+[#Katana]
```
Comment (stoobie) [inline modules/ROOT/partials/partial_devtools.adoc]: Please leave the id as-is. Id's impact links, so if you change an id, you have to check to make sure that there aren't any existing links to the id that also need to change.

This is not visible, except possibly in URLs, which we in any case want to leave as lower-case.

**Review by stoobie — CHANGES_REQUESTED** [review]
Other than my comment about the id, we can't mention Katana without also mentioning starknet-devnet-rs.

**Comment by LandauRaz** [comment]
Added as part of [#1332](https://github.com/starknet-io/starknet-docs/pull/1332). Thanks for all your help, @gianalarcon!

**Pull request closed without merging**